### DISCUSSION TITLE
Is there any way to add comments to the generated rbac?

### DISCUSSION THREAD

**Discussion by joejulian** [kubernetes-sigs/kubebuilder#3649] [Support] [answered]
I'm pretty sure there's not, but it would be really nice to have something like
```
//+kubebuilder:rbac:groups=core,resources=nodes,verbs=get;list;watch,comment="this rule is needed to watch for nodes being deleted"
```
Is there any other way to accomplish this?

**Comment by camilamacedo86** [comment, marked as answer]
Today, it is not possible.
If you we would like to achieve this could you please open an issue / RFE to the controller-tools project: https://github.com/kubernetes-sigs/controller-tools ?

This is the project responsible for the controller-gen
